### DISCUSSION TITLE
Crystal Orientation

### DISCUSSION THREAD

**Discussion by ccis88** [neperfepx/neper#670] [Q&A] [answered]
In Tessellation Module stage, a random orientation was selected for the grain. After exporting the STL file format and importing it into PZFLEX, if all the grains are assigned the same material, is the grain orientation still there? Or is it equivalent to a complete grain-independent model that gives the same material all grains?
Thanks for the answer！！

**Comment by rquey** [comment, marked as answer]
As it stands, the orientations are not written to the STL file...

**Reply by ccis88** [reply]
Thank you for your answer. I also want to ask, can rotate and hspace be used at the same time? I couldn't do it all the time
